tests/test_concrete_not_packed_elf32_arm.py

```diff
@@ -0,0 +1,85 @@
+import angr
+import avatar2
+import claripy
+import nose
+import os
+
+from angr_targets import AvatarGDBConcreteTarget
+
+GDB_SERVER_IP = '192.168.60.3'
```

Comment:
How will this work on CI?

Reply:
I've removed the hardcoded IP, but anyway, this test won't actually run on CI.